PacketDefinitions420/PacketDefinitions/S2C/BuyItemResponse.cs

```diff
@@ -10,7 +10,7 @@ public BuyItemResponse(IAttackableUnit actor, IItem item, byte unk = 0x29)
             : base(PacketCmd.PKT_S2C_BUY_ITEM_ANS, actor.NetId)
         {
             Write((int)item.ItemData.ItemId);
-            Write((byte)actor.Inventory.GetItemSlot(item));
+            Write((byte)((IObjAiBase)actor).Inventory.GetItemSlot(item));
```

Comment:
If you have time, it may be nice if you switched this to use LeaguePackets (where this packet is used is NotifyBuyItem and NotifyItemBought, this is where you would implement LeaguePackets' version). Take a look at NotifyMinionSpawned for some insight on how it should be structured.

Reply:
Sure I'll give it a go.